src/modules/about/Team.js

```diff
@@ -0,0 +1,182 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Tout from 'components/Tout';
+import mq from '../../utils/mq';
+import frances from 'media/frances.jpg';
+import john from 'media/john.jpg';
+import jesse from 'media/jesse.jpg';
+
+const Container = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  border-bottom: 2px solid white;
+  ${mq.desktop`
+    border-bottom: 5px solid white;
+    flex-direction: row;
+    padding: ${props => props.theme.paddingDesktop};
+  `}
+  ${mq.tablet`
+    padding: ${props => props.theme.paddingTablet};
+  `}
+  ${mq.mobile`
+    flex-direction: column;
+    padding: ${props => props.theme.paddingMobile};
+  `}
+`;
+
+const InfoColumn = styled.div`
+  ${mq.desktop`
+    width: calc(100%/3 - 35px);
```

Comment:
Pedantic but it's easier to read this spaces here.
```suggestion
    width: calc(100% / 3 - 35px);
```